README.md

```diff
@@ -28,30 +59,35 @@ And after that:
 [GET] api/sitemaps/generate?storeId=...&baseUrl=...&sitemapUrl=...
 ```
 
-# Documentation
-User guide: [Sitemaps](http://virtocommerce.com/docs/vc2userguide/sitemaps)
+## Installation
 
-# Installation
 Installing the module:
-* Automatically: in VC Manager go to Configuration -> Modules -> Sitemaps module -> Install
-* Manually: download module zip package from https://github.com/VirtoCommerce/vc-module-sitemaps/releases. In VC Manager go to Configuration -> Modules -> Advanced -> upload module package -> Install.
+1. Automatically: in VC Manager go to More -> Modules -> Sitemaps module -> Install;
+
+1. Manually: download module zip package from https://github.com/VirtoCommerce/vc-module-sitemaps/releases. In VC Manager go to More -> Modules -> Advanced -> upload module package -> Install.
+
+## Settings
+
+Virto Commerce Sitemaps module has several settings, each of them influences the sitemap XML generation process. The settings are grouped into sections.
 
-# Settings
 ### General settings
-* **Records limit** (default value: **10000**) - sets the maximum number of URLs record per sitemap file
-* **Filename separator** (default value: **--**) - sets the sitemap location separator in case of sitemap items number exceeds the **Records limit** parameter value (i.e.: "products.xml" -> "products--1.xml" and "products--2.xml")
-* **Search bunch size** (default value: **1000**) - this parameter is using in long-term search processes (i.e. catalog search) to divide search requests and sets the search request bunch size parameter
-* **Export/Import description** (default value: **Export/Import sitemaps with all sitemap items**) - sets the description for platform export/import process
+
+1. Records limit** (default value: **10000**) - sets the maximum number of URLs record per sitemap file;
```

Comment:
What does `**` in `Records limit**` mean?

Reply:
simply an error, will delete it!